### ISSUE TITLE
xD

### ISSUE THREAD

**Issue by ImagicalGamer** [ImagicalGamer/Pig#1]
@Taki21 @remotevase I need halp xD. You can see the code in Pig.php but dis is da error:
`[15:11:13] critical> Unhandled exception executing command 'pig remove SwaggyHUD' in pig: Argument 1 passed to pocketmine\plugin\PluginManager::disablePlugin() must implement interface pocketmine\plugin\Plugin, string given, called in C:\Users\Jake\Downloads\ImagicalMine\plugins\Pig-master\src\ImagicalGamer\Pig\Pig.php on line 33
[15:11:13] critical> TypeError: "Argument 1 passed to pocketmine\plugin\PluginManager::disablePlugin() must implement interface pocketmine\plugin\Plugin, string given, called in C:\Users\Jake\Downloads\ImagicalMine\plugins\Pig-master\src\ImagicalGamer\Pig\Pig.php on line 33" (EXCEPTION) in "/src/pocketmine/plugin/PluginManager" at line 648
`

**Comment by ImagicalGamer** [comment]
It implements Plugin idk whats wrong

**Comment by ImagicalGamer** [comment]
You guys r the only ppl who can handle my nooby-ness sometimes xD

**Comment by Taki21** [comment]
How about now?

**Comment by ImagicalGamer** [comment]
lol didn't fix anything same error.

**Comment by Taki21** [comment]
AAH

**Comment by Taki21** [comment]
Sam error?

**Comment by ImagicalGamer** [comment]
Yep

**Comment by Taki21** [comment]
Thats weird i removed PluginManager!

**Comment by remotevase** [comment]
sorry dude, I can't help you with noobness, as I often have mistakes like that myself. I usually call on @Taki21 and @applqpak

**Comment by ImagicalGamer** [comment]
I got it xD

**Comment by Taki21** [comment]
really @remotevase ? GO TYPE SOME PHP ESSAYS! xD

**Comment by ImagicalGamer** [comment]
Okie, now all I need to do is figure out how to get the download from the forums, or upload ALL plugs onto a web server xD.

**Comment by ImagicalGamer** [comment]
@taki21 @remotevase wouldn't the forum plugins be stored somewhere on the forum server? Kinda like a cloud server of some sort?

**Comment by ImagicalGamer** [comment]
@applqpak ^^

**Comment by applqpak** [comment]
What?

**Comment by ImagicalGamer** [comment]
How would I get the data from the forums? Or I can upload all plugs onto a web server :smile:.

**Comment by Taki21** [comment]
What do ya mean forum server?

**Comment by ImagicalGamer** [comment]
I meant the plugins on the forums, they have to be stored somewhere.

**Comment by keithkfng** [comment]
Yes they are, there all stored in an encrypted format which I can't decode. Sorry.

**Comment by applqpak** [comment]
Lol

**Comment by ImagicalGamer** [comment]
lel

**Comment by ImagicalGamer** [comment]
A web server it is xP.

**Comment by ImagicalGamer** [comment]
LOL I have spent hours screwing with a plug trying to figure out why it wouldn't send a message to the command sender when I had everything right and I realized my server is corrupt.

**Comment by ImagicalGamer** [comment]
Is PlayerJoinEvent broken? It won't let me do anything with it. No errors or notices, code is fine. Just doesn't do it.
